cmd/ak/cmd/vars/set.go

```diff
@@ -28,7 +31,20 @@ var setCmd = common.StandardCommand(&cobra.Command{
 			return fmt.Errorf("invalid variable name %q: %w", args[0], err)
 		}
 
-		ev := sdktypes.NewVar(n, args[1], secret).WithScopeID(id)
+		var value string
+		if len(args) == 1 {
```

Comment:
Optional nit: given that `== 2` is the "trivial" case, and it takes only 1 line of code, it's easier to read the cade if you reverse the order of the branches.

Reply:
Done.